lotto/src/main/kotlin/lotto/domain/Lotto/Lotto.kt

```diff
@@ -0,0 +1,46 @@
+package lotto.domain.Lotto
+
+import lotto.domain.result.JudgeResult
+
+class Lotto(
+    val numbers: Set<Number>
+) {
+
+    fun judgeWinning(winningLotto: Lotto, bonusNumber: Number): JudgeResult {
+        var sameCount = 0
+        var bonusWin = false
+
+        for (number in numbers) {
+            if (winningLotto.contains(number)) {
+                sameCount += 1
+            }
+        }
+
+        if (numbers.contains(bonusNumber)) {
+            bonusWin = true
+        }
```

Comment:
```suggestion
        val sameCount = numbers.count { winningLotto.numbers.contains(it) }
        val bonusWin = numbers.contains(bonusNumber)
```
코드를 매우 간결하게 줄일 수 있을 것 같아요 :)

Reply:
크 이런 내장 함수가 아직 안익숙한데 감사합니다